sema/syncfs/service.py

```
@@ -151,15 +168,15 @@ def perform_sync(from_path: Path, to_store: RDFStore) -> None:
             log.debug(f"skip file {fname} with lastmod {lastmod} - unchanged")
 
 
-class SyncFsTriples:
+class SyncFsTriples(ServiceBase):
     """Process-wrapper-pattern for easy inclusion in other contexts."""
 
     def __init__(
```

Ruff

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)